py-polars/polars/lazyframe/frame.py

```diff
@@ -1936,9 +1960,18 @@ def collect(
             issue_unstable_warning("Background mode is considered unstable.")
             return InProcessQuery(ldf.collect_concurrently())
 
-        # Only for testing purposes atm.
+        if _eager:
+            # Don't run on GPU in _eager mode (but don't warn)
+            gpu = False
+        callback = None
+        if gpu:
+            cudf_polars = import_optional(
+                "cudf_polars", err_prefix="GPU engine requested, but required_package"
```

Comment:
Two suggestions: replace an underscore with a space, and add -cu12 / -cu11 suffixes to the install command. Alternatively we could link to an installation guide.
```suggestion
                module_name="cudf_polars",
                err_prefix="GPU engine requested, but required package",
                install_message="Please install using the command `pip install cudf-polars-cu12` or `pip install cudf-polars-cu11`",
```

Reply:
We haven't fully pinned down the install setup yet.